dashboard/app/controllers/application_controller.rb

```diff
@@ -325,6 +328,24 @@ def clear_sign_up_session_vars
     end
   end
 
+  # Check that the user is compliant with the Child Account Policy. If they
+  # are not compliant, then we need to send them to the lockout page.
+  def assert_child_account_policy
+    # Check that the child account policy is currently enabled.
+    return unless CPA.cpa_experience(request)
+
+    # Anonymous users are affected by our Child Account Policy
+    return unless current_user
+    # Don't block any user from signing out
+    return if request.path == '/users/sign_out'
+    # Don't block any user from changing the language
+    return if request.path == '/locale'
+    # Avoid an infinite redirect loop to the lockout page
+    return if request.path == '/lockout'
```

Comment:
I could have done this with a single if statement, but having it broken up like this lets me add comments/context for each URL. Let me know what you think.

Reply:
or something like: (but it's not really improving things a whole lot)

```
  return if [
    "/users/sign_out", # Don't block any user from signing out
    "/locale",         # Don't block any user from signing out
    "/lockout",        # Avoid an infinite redirect loop to the lockout page
  ].include? request.path`

Reply:
done